docs/_documentation/plugins/ios-sidebar.md

```diff
@@ -4,6 +4,19 @@ title: iOS Sidebar
 category: Plugins
 ---
 
+## Setup
+
+You must override `CreateViewPresenter` in your `MvxIosSetup` class too tell MvvmCross to use the `MvxSidebarPresenter` instead of the default [iOS ViewPresenter](https://www.mvvmcross.com/documentation/platform/ios/ios-view-presenter?scroll=678).
```

Comment:
```suggestion
You must override `CreateViewPresenter` in your `MvxIosSetup` class too tell MvvmCross to use the `MvxSidebarPresenter` instead of the default [iOS ViewPresenter](https://www.mvvmcross.com/documentation/platform/ios/ios-view-presenter).
```